src/components/CortexPage/CortexPage.tsx

```diff
@@ -65,11 +72,16 @@ export const CortexPage = ({
       Show the settings page if there is an error (will occur when email header authorization returns an error, which is a valid use case)
       or if the user has the EDIT_SETTINGS permission
        */}
-      {!hideSettings && (!isUndefined(error) || canEditSettings) && (
+      {!hideSettings && (!isUndefined(permissionsError) || canEditSettings) && (
         <CortexLayout.Route path="settings" title="Settings">
           <SettingsPage />
         </CortexLayout.Route>
       )}
+      {!isUndefined(helpPage) && (
```

Comment:
we should prefer isNil because it covers both null & undefined
```suggestion
      {!isNil(helpPage) && (
```